server/db/schema.sql

```diff
@@ -17,4 +21,20 @@ CREATE TABLE adoptable_pets (
   adoption_story VARCHAR(255) NOT NULL,
   available_for_adoption BOOLEAN DEFAULT TRUE,
   pet_type_id INTEGER REFERENCES pet_types(id)
-);
\ No newline at end of file
+<<<<<<< HEAD
+);
+
+
+CREATE TABLE adoption_applications(
+  id SERIAL PRIMARY KEY,
+  name VARCHAR(255) NOT NULL,
+  phone_number VARCHAR(255) NOT NULL,
+  email VARCHAR(255) NOT NULL,
+  home_status VARCHAR(255) NULL NULL,
+  application_status VARCHAR(255) DEFAULT 'Pending' NOT NULL,
+  adoptable_pet_id INTEGER REFERENCES adoptable_pets(id) 
+);
+
+=======
+);
+>>>>>>> 4de76760c73ebdc4e390070a9d06f7d0dd57dabf
```

Comment:
merge conflict lines on 38, 40 and maybe even 39?  I would ensure the schema file works correctly and everything is closed off properly after making the changes above.